pages/plugins/writing.md.erb

```diff
@@ -46,10 +49,10 @@ configuration:
 
 The `configuration` property defines the validation rules for the plugin configuration using the [JSON Schema](https://json-schema.org) format. The plugin in this tutorial has a single `pattern` property, of type `string`.
 
-Configuration properties are available to the hook script as environment variables with the naming pattern `BUILDKITE_PLUGIN_<PLUGIN_NAME>_<CONFIGURATION_PROPERTY>` where `<PLUGIN_NAME>` is the GitHub repository slug, not the name defined in `plugin.yml`. In this case, the configured value of `pattern` will be available as `BUILDKITE_PLUGIN_FILE_COUNTER_PATTERN`.
+Configuration properties are available to the hook script as environment variables with the naming pattern `BUILDKITE_PLUGIN_<PLUGIN_NAME>_<CONFIGURATION_PROPERTY>` where `<PLUGIN_NAME>` is not the name defined in `plugin.yml` but an uppercase version of the plugin's ID. In this case, the configured value of `pattern` will be available as `BUILDKITE_PLUGIN_FILE_COUNTER_PATTERN`.
 
->🚧 Plugin properties and git
-> Note that if you <a href="/docs/plugins/using#plugin-sources">reference a plugin</a> using a full URL and <code>.git</code> extension, you need to use a slightly different syntax to get access to the configuration properties. For example, to get the value of <code>pattern</code> in <code>https://github.com/my-org/my-plugin.git#v1.0.0</code> use  <code>BUILDKITE_PLUGIN_MY_PLUGIN_GIT_PATTERN</code>.
+>🚧 Variables and non-standard git plugins
```

Comment:
The rest of the page frames these ideas in terms of _properties_ rather than _variables_, so I think it would be good to stick to that. I think it could be more descriptive. Thinking:

```suggestion
>🚧 Accessing plugin properties with Git URLs
```

(_Git_ should also be capitalized)

Reply:
How about the follwing to avoid a potential confusion of Git URLs being the values of the plugin properties?

```suggestion
>🚧 Accessing properties on plugins referenced with Git URLs
```